docs/usage.md

```diff
@@ -144,6 +144,21 @@ Then pass this to `osv-scanner` with this:
 osv-scanner --lockfile osv-scanner:/path/to/osv-scanner.json
 ```
 
+## Specify Package(s) to Exclude from OSV Query
+
+If you want to exclude a package or a range of packages from the query to the
+OSV database, you can pass a list of regex patterns with the '--exclusions'
+flag. The `osv-scanner` will check for packages matching the specified regex
```

Comment:
```suggestion
flag. OSV-Scanner will check for packages matching the specified regex
```
This phrasing is more consistent with our documentation.

Reply:
Sure thing!